expressionvalidity.c

```diff
@@ -0,0 +1,95 @@
+#include<stdio.h>
+#define size 10
+#define TRUE 1
+#define FALSE 0
+struct stack
+{
+    int top;
+    char Item[size];
+};
+struct stack s;
+int Intialize(void)
+{
+    s.top=-1;
+}
+ int StackTop()
+ {
+     int x;
+     x=s.Item[S.top];

```

Comment:
What is X?

Reply:
do i need to the variable description or has to change the variables ?